src/ui/components/MarkdownPart/MarkdownPart.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { MarkdownPartProps } from 'features/offer/types'
+import { highlightLinks } from 'libs/parsers/highlightLinks'
+import { TypoDS } from 'ui/theme'
+
+export const MarkdownPart: FunctionComponent<MarkdownPartProps> = ({
+  text,
+  isBold,
+  isItalic,
+  isUnderline,
+}) => {
+  if (isBold && !isItalic) {
+    return (
+      <StyledBodyAccent isUnderline={isUnderline} testID="styledBodyAccent">
+        {highlightLinks(text)}
+      </StyledBodyAccent>
+    )
+  } else if (isItalic && !isBold) {
+    return (
+      <StyledBodyItalic isUnderline={isUnderline} testID="styledBodyItalic">
+        {highlightLinks(text)}
+      </StyledBodyItalic>
+    )
+  } else if (isItalic && isBold) {
+    return (
+      <StyledBodyItalicAccent isUnderline={isUnderline} testID="styledBodyItalicAccent">
+        {highlightLinks(text)}
+      </StyledBodyItalicAccent>
+    )
+  }
+
+  return (
+    <StyledBody isUnderline={isUnderline} testID="styledBody">
+      {highlightLinks(text)}
+    </StyledBody>
+  )
+}
+
+const StyledBody = styled(TypoDS.Body)<{ isUnderline?: boolean }>(({ isUnderline }) => ({
+  textDecorationLine: isUnderline ? 'underline' : 'none',
+}))
```

Comment:
la fonction de style me semble etre répétée à l'identique, on pourrait l'extraire (j'ai pas d'idée de nom)

```suggestion
const underlineStyle = ({ isUnderline }) => ({
  textDecorationLine: isUnderline ? 'underline' : 'none',
})

const StyledBody = styled(TypoDS.Body)<{ isUnderline?: boolean }>(underlineStyle)
```

Reply:
Du coup je l'ai carrément supprimée car on ne s'occupe pas de souligné pour le moment.